tests/integration-tests/tests/nccl/test_nccl/test_nccl/nccl_tests_submit_openmpi.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#SBATCH --nodes=2
+#SBATCH --exclusive
+#SBATCH --ntasks-per-node=4
```

Comment:
I think this is the place we are forcing the use of 4 GPUs, by not allocating all CPUs for the test?

In any case, it would be nice to try to use all 8 GPUs